src/FixedDataTableCellGroup.js

```diff
@@ -227,66 +251,4 @@ class FixedDataTableCellGroupImpl extends React.Component {
   };
 }
 
-class FixedDataTableCellGroup extends React.Component {
-  /**
-   * PropTypes are disabled in this component, because having them on slows
-   * down the FixedDataTable hugely in DEV mode. You can enable them back for
-   * development, but please don't commit this component with enabled propTypes.
-   */
-  static propTypes_DISABLED_FOR_PERFORMANCE = {
-    isScrolling: PropTypes.bool,
-    /**
-     * Height of the row.
-     */
-    height: PropTypes.number.isRequired,
-
-    offsetLeft: PropTypes.number,
-
-    left: PropTypes.number,
-    /**
-     * Z-index on which the row will be displayed. Used e.g. for keeping
-     * header and footer in front of other rows.
-     */
-    zIndex: PropTypes.number.isRequired,
-  };
-
-  shouldComponentUpdate(/*object*/ nextProps) /*boolean*/ {
-    /// if offsets haven't changed for the same cell group while scrolling, then skip update
-    return !(
-      nextProps.isScrolling &&
-      this.props.rowIndex === nextProps.rowIndex &&
-      this.props.left === nextProps.left &&
-      this.props.offsetLeft === nextProps.offsetLeft
-    );
-  }
-
-  static defaultProps = /*object*/ {
-    left: 0,
-    offsetLeft: 0,
-  };
-
-  render() /*object*/ {
-    var { offsetLeft, ...props } = this.props;
-
-    var style = {
-      height: props.cellGroupWrapperHeight || props.height,
-      width: props.width,
-    };
-
-    if (this.props.isRTL) {
-      style.right = offsetLeft;
-    } else {
-      style.left = offsetLeft;
-    }
-
-    return (
-      <div
-        style={style}
-        className={cx('fixedDataTableCellGroupLayout/cellGroupWrapper')}
-      >
-        <FixedDataTableCellGroupImpl {...props} />
-      </div>
-    );
-  }
-}
 export default FixedDataTableCellGroup;
```

Comment:
Now there is no need of this wrapper div .We have provided the styling and the className to the corresponding div .So we can remove this FixedDataTableCellGroup class component and rename FixedDataTableCellGroupImpl to FixedDataTableCellGroup as we are passing all the props from FixedDataTableRow to FixedDataTableCellGroup.